data/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <script src="screenly.js"></script>
+    <title>Hello world</title>
+  </head>
+  <body>
+    <p id="message">Hello world from <span id="hostname-placeholder">%1</span></p>
```

Comment:
%1 is not needed here.

Reply:
are you sure? I tested it on the device %1 was not displayed